helix-term/src/ui/mod.rs

```diff
@@ -185,6 +186,20 @@ pub fn raw_regex_prompt(
     cx.push_layer(Box::new(prompt));
 }
 
+pub fn get_relative_dir(path: &Path) -> Cow<'static, str> {
+    let path = helix_stdx::path::get_relative_path(path);
+    if path.components().next().is_none() {
+        "./".into()
+    } else {
+        let mut str = path.to_string_lossy().into_owned();
+        if path.is_dir() {
+            str.push('/');
+        }
+
+        str.into()
+    }
+}
```

Comment:
isn't this always called with a directory as entry (`&root`) ? Could we dispense the FS check (slow) then ?

Reply:
Makes sense, done.